src/coreComponents/linearAlgebra/interfaces/hypre/mgrStrategies/CompositionalMultiphaseHybridFVM.hpp

```diff
@@ -70,15 +70,25 @@ class CompositionalMultiphaseHybridFVM : public MGRStrategyBase< 3 >
 
     setupLabels();
 
-    m_levelFRelaxMethod[0] = 0; // Jacobi
-    m_levelFRelaxMethod[1] = 0; // Jacobi
-    m_levelFRelaxMethod[2] = 0; // Jacobi
-
-    m_levelInterpType[0] = 2;
-    m_levelInterpType[1] = 2;
-    m_levelInterpType[2] = 2;
-
-    m_globalSmoothType = 16; // ILU(0)
+    // Level 0
+    m_levelFRelaxMethod[0]     = hypre::MGRFRelaxationMethod::singleLevel; //default, i.e. Jacobi (to be confirmed)
```

Comment:
You can drop `hypre::` on these (if you want) since we're already inside `hypre` namespace